internal/pkg/build/stage.go

```diff
@@ -83,8 +83,8 @@ func (s *stage) runHostScript(name string, script types.Script) error {
 
 func (s *stage) runPostScript(sessionResolv, sessionHosts string) error {
 	if s.b.Recipe.BuildData.Post.Script != "" {
-		cmdArgs := []string{"-s", "--build-config", "exec", "--pwd", "/", "--writable"}
-		cmdArgs = append(cmdArgs, "--cleanenv", "--env", aEnvironment, "--env", sEnvironment, "--env", aLabels, "--env", sLabels)
+		cmdArgs := []string{"--build-config", "exec", "--pwd", "/", "--writable"}
+		cmdArgs = append(cmdArgs, "--cleanenv", "--env", aEnvironment, "--env", sEnvironment, "--env", aLabels, "--env", sLabels, "--env", "APPTAINER_MESSAGELEVEL="+sylog.GetEnvVar())
```

Comment:
This might be overkill since normally the command line message level setting is passed on to the nested apptainer.  I guess if someone set the environment variable instead of the command line, this would help.  But I'm worried about extraneous INFO messages in the default condition of no messagelevel settings or command line options.  For example what happens if you don't have subuid mapping set up, are there any INFO messages shown again regarding fakeroot?

Reply:
thanks for the good point, yeah, we do not need to pass the env variable as you said. It works as the same as passing the env variable. I also test with `--ignore-subuid` and INFO/WARNING messages are also shown as well. see test
https://github.com/apptainer/apptainer/pull/2556#issue-2574612173

Reply:
Ok, so the INFO message about no subuid does not appear twice.  So I guess it's OK to leave it at the default INFO level instead of dropping it to WARNING by default.